detox/src/devices/Fbsimctl.js

```diff
@@ -173,6 +173,14 @@ class Fbsimctl {
     await this._execFbsimctlCommand(options);
   }
 
+  async resetContentAndSettings(udid) {
```

Comment:
`resetContentAndSettings` is fine for a name.
I don’t feel comfortable with having UDID of the sim in the test? Maybe it should be `device.resetContentAndSettings()` and that will reset the current device.